en/configure-a-tidb-cluster.md

```diff
@@ -196,6 +196,28 @@ spec:
       cpu: 1
 ```
 
+Since TiDB Operator v1.1.6, support passing raw TOML config for the component:
```

Comment:
```suggestion
Since v1.1.6, TiDB Operator supports passing raw TOML configuration to the component:
```
@july2993 Please confirm whether this is correct.